### ISSUE TITLE
Syntax highlighting issue with comment blocks

### ISSUE THREAD

**Issue by Dubz** [ahkscript/SublimeAutoHotkey#54]
If you add comment blocks using /*...*/ and the closing */ is not at the beginning of a new line, the highlighting makes everything after it appeared commented out.

**Comment by Dubz** [comment]
I opened a file and screenshotted it for reference.

Not closing at the beginning of a new line
![image](https://user-images.githubusercontent.com/2218993/60617509-e5ab1200-9da1-11e9-8f3c-ca42f5800a53.png)

Closing at the beginning of a new line.
![image](https://user-images.githubusercontent.com/2218993/60617603-0f643900-9da2-11e9-8661-66062d30fa36.png)

**Comment by robertcollier4** [comment]
See:
https://www.autohotkey.com/docs/Language.htm#comments
In addition, the /* and */ symbols can be used to comment out an entire section, but only if the symbols appear at the beginning of a line as in this example:

**Comment by Dubz** [comment]
This is not entirely true, though. I just made a simple script with the whitespace included before the closing comment tag (` */`), and it ran just fine. Tested with multiple spaces, and tabs. All work no problem.
Anything else before the closing `*/` tag, apart from whitespace, is not allowed/doesn't work. This is including an extra asterisk (ex. `**/`)

See below for example:
```
/**
 * This is a comment block test
 *
 * Loreum ipsum yada yada
    		    */

MsgBox, Hello world!
```

Add any character that is not a space or tab before `*/`, and the message box won't show when ran.
